docs/guides/first-device-level-experiment.mdx

```diff
@@ -3,62 +3,84 @@ sidebar_label: Device-level Experiments
 title: Build your first Device-level Experiment
 ---
 
-In cases where you're unable to establish the user's identity and are unable to leverage a user ID as the unit of randomization for your experiment,
-Statsig's _device-level_ experiments enable you to design experiments using a Statsig-generated stable ID, saved on the user's device in local storage.
+When you cannot identify a user via their user ID, device-level experiments allow you to randomize experiments based on a _stable ID_ stored locally on the user's device. This stable ID is automatically generated by Statsig and ensures consistent evaluations across sessions for anonymous users.
```

Comment:
I feel like we should push people to use their own cookie/logged out id, and only rely on stableid if they have nothing